### PR TITLE
Merge updates to master

### PR DESCRIPTION
**Pull request by m1cr0man**
Separated out the mailman stuff. Includes the other mail stuff for a fully functioning postfix + dovecot setup.

### CODE REVIEW

**Comment by m1cr0man** [comment]
Please note: Use `nixos-rebuild boot` and reboot hardcase since I changed the postgres data path

File: services/httpd/vhosts.nix [hunk 1]
```diff
@@ -3,287 +3,251 @@ with (import ./shared.nix { tld = config.redbrick.tld; });
...
     documentRoot = "${webtree}/r/receive/abovethefold";
     user = "receive";
     group = "staff";
-  })
-  (vhost {
-    hostName = "blog.${tld}";
+    wwwRedirect = true;
+    serverAliases = [ "www.abovethefold.es" ];
+  };
+  "blog.${tld}" = vhost {
```
Comment (butlerx) [inline services/httpd/vhosts.nix:32]: so I think it'd be a good idea if we alphabetise this list. https://github.com/redbrick/nix-configs/blob/blog/services/httpd/vhosts.nix <- was done in the testing blog so the only difference is the lack of blog

File: services/httpd/vhosts.nix [hunk 2]
```diff
@@ -3,287 +3,251 @@ with (import ./shared.nix { tld = config.redbrick.tld; });
...
     documentRoot = "${webtree}/a/ainran/mlane";
     user = "ainran";
     group = "associat";
-  })
-  (vhost {
-    hostName = "obrienronan.com";
+    wwwRedirect = true;
+    serverAliases = [ "www.mlane.org" ];
+  };
+  "nemo.${tld}" = vhost {
```
Comment (butlerx) [inline services/httpd/vhosts.nix]: this was deleted from main

File: services/httpd/vhosts.nix [hunk 3]
```diff
@@ -292,237 +256,206 @@ in (if (config.redbrick.skipVhosts) then [] else ([
...
     documentRoot = "${webtree}/vhosts/www.songsfromtheparlour.com";
     user = "parlour";
     group = "guest";
-  })
-  (vhost {
-    hostName = "speakeasyireland.ie";
+    wwwRedirect = true;
+    serverAliases = [ "www.songsfromtheparlour.com" ];
+  };
+  # TODO dead
```
Comment (butlerx) [inline services/httpd/vhosts.nix]: delete?

**Review by butlerx — CHANGES_REQUESTED** [review]

**Comment by butlerx** [comment]
let's send out a maintenance notice for the reboot

File: services/httpd/vhosts.nix (hunk 1 above)
Comment (m1cr0man) [reply services/httpd/vhosts.nix:32]: Yeah fair enough. Shall we not wait until the blog pr is merged then?

**Comment by m1cr0man** [comment]
I actually deleted 3 vhosts. I'll reintroduce that patch

File: services/httpd/vhosts.nix (hunk 1 above)
Comment (butlerx) [reply services/httpd/vhosts.nix:32]: sure it can wait till #38

**Review by butlerx — APPROVED** [review]

**Pull request merged**